### ISSUE TITLE
Research additional datasets and ingest them (starting with Bavaria)

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#122]
## User story
1. As a User
2. I want more data about a specific lot (or coordinate inside a lot) (e.g.: Parzellar-Karte, Tatsaechliche Nutzung, Hausumringe, 3D-LoD2, DMG, etc.)
4. So that I can have more information about a certain coordinate

## Acceptance criteria
* [ ] Ingest (specific) data from: https://geodaten.bayern.de/opengeodata/index.html
* [ ] Reasearch and ingest further datasources (opengeodata) for other states beside bavaria
* [ ] Document sources
* [ ] Create UI views for according datasets (or just adopt existing ones)
* [ ] Create API points

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Corgam** [comment]
Removed from product backlog by POs. Closing.
